hw/opentitan/ot_spi_device.c

```diff
@@ -252,64 +250,67 @@ REG32(TPM_WRITE_FIFO, 0x38u)
  */
 #define SPI_BUS_FLASH_READ_DELAY_NS 100000000u
 
-/*
- *          New scheme (Egress + Ingress)      Old Scheme (DPSRAM)
- *         +-----------------------------+    +-----------------------+
- *         | Flash / Passthru modes      |    | Flash / Passthru modes|
- *  0x000 -+----------------+------+-----+   -+----------------+------+
- *         | Read Command 0 | 1KiB | Out |    | Read Command 0 | 1KiB |
- *  0x400 -+----------------+------+-----+   -+----------------+------+
- *         | Read Command 1 | 1KiB | Out |    | Read Command 1 | 1KiB |
- *  0x800 -+----------------+------+-----+   -+----------------+------+
- *         | Mailbox        | 1KiB | Out |    | Mailbox        | 1KiB |
- *  0xc00 -+----------------+------+-----+   -+----------------+------+
- *         | SFDP           | 256B | Out |    | SFDP           | 256B |
- *  0xd00 -+----------------+------+-----+   -+----------------+------+
- *         |                             |    | Payload FIFO   | 256B |
- *  0xe00 -+----------------+------+-----+   -+----------------+------+
- *         | Payload FIFO   | 256B | In  |    | Command FIFO   |  64B |
- *  0xe40 -+----------------+------+-----+   -+----------------+------+
- *         | Command FIFO   |  64B | In  |    | Address FIFO   |  64B |
- *  0xe80 -+----------------+------+-----+   -+----------------+------+
- *         | Address FIFO   |  64B | In  |
- *  0xe80 -+----------------+------+-----+
+/* Memory layout extracted from the documentation:
```

Comment:
Nit: QEMU multi-line comment style
```suggestion
/*
 * Memory layout extracted from the documentation:
```